config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -54,6 +54,7 @@ periodics:
       - --
       - --cluster=gce-scale-cluster
       - --env=HEAPSTER_MACHINE_TYPE=n1-standard-32
+      - --env=DNS_MEMORY_LIMIT=300Mi
```

Comment:
Can you add a TODO to adjust it (or remove) once we fix the regression?

Reply:
Done